openhtf/io/frontend/templates/test.html

```diff
@@ -12,6 +12,17 @@
 
 </div>
 {% else %}
+  <h2>{{state['test']['record']['dut_id']}}</h2>
+  {% if state['test']['status'] == 'PASS' %}
+    <h1 style="color:green;">
+  {% elif state['test']['status'] == 'FAIL' %}
+    <h1 style="color:red;">
+  {% else %}
+    <h1 style="color:orange;">
+  {% end %}
+  {{state['test']['status']}}</h1>
+
+{% if state['framework']['status'] == 'EXECUTING' %}
```

Comment:
Downside of tornado templating versus flask. Their {{}} expressions are basically python, which means the data we're passing in is python dicts. This could be avoided by just moving the raw json around instead of parsing it, but since the current frontend server does some of the logic here we parse. The new frontend will have more logic built into the client, so the server will be able to just feed it json state updates from framework instances.